### PR TITLE
Normalise phpdoc

### PR DESCRIPTION
**Pull request by GrahamCampbell**
Enables some non-controversial phpdoc fixers.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Question: do we also want to enable alignment of phpdoc annotations? The current diff takes some out of alignment... however enforcing alignment is likely to change A LOT. ;)

File: src/Faker/Provider/Barcode.php [hunk 1]
```diff
@@ -39,7 +39,7 @@ protected static function eanChecksum($input)
      * @param  string           $input ISBN without check-digit
      * @throws \LengthException When wrong input length passed
      *
-     * @return integer Check digit
+     * @return int Check digit
```
Comment (Nyholm) [inline src/Faker/Provider/Barcode.php:42]: Should be `int|string`

File: src/Faker/Provider/me_ME/Payment.php [hunk 2]
```diff
@@ -9,7 +9,7 @@ class Payment extends \Faker\Provider\Payment
      * @link http://en.wikipedia.org/wiki/International_Bank_Account_Number
      * @param  string  $prefix      for generating bank account number of a specific bank
      * @param  string  $countryCode ISO 3166-1 alpha-2 country code
-     * @param  integer $length      total length without country code and 2 check digits
+     * @param  int $length      total length without country code and 2 check digits
      * @return string
      */
     public static function bankAccountNumber($prefix = '', $countryCode = 'ME', $length = '18')
```
Comment (Nyholm) [inline src/Faker/Provider/me_ME/Payment.php:15]:
```suggestion
    public static function bankAccountNumber($prefix = '', $countryCode = 'ME', $length = 18)
```

**Review by Nyholm — COMMENTED** [review]
Thank you. Just make sure CI is green.

**Comment by Nyholm** [comment]
I dont really care if they are aligned or not. Let's do whatever produces the smallest diff.

File: src/Faker/Provider/File.php [hunk 3]
```diff
@@ -566,7 +566,7 @@ public static function fileExtension()
      *
      * @param  string  $sourceDirectory The directory to look for random file taking
      * @param  string  $targetDirectory
-     * @param  boolean $fullPath        Whether to have the full path or just the filename
+     * @param  bool $fullPath        Whether to have the full path or just the filename
```
Comment (Nyholm) [inline src/Faker/Provider/File.php:569]: They are currently aligned. Let's enforce that then.

**Comment by GrahamCampbell** [comment]
Maybe will do a follow-up to see what that looks like.

**Comment by GrahamCampbell** [comment]
Gahhh

**Comment by Nyholm** [comment]
Hehe, how is it going?

**Comment by GrahamCampbell** [comment]
Enabling separation separates too much, and enabling alignment aligns params and returns together, which leads to two spaces after param... not what we want.

**Comment by GrahamCampbell** [comment]
I think we can just not care about alignment for now, until we have cleaned up the codebase to a point where separation can be enabled.

**Review by Nyholm — APPROVED** [review]

**Pull request merged**